zng/docs/spec.md

```diff
@@ -652,12 +680,13 @@ Type | Format
 `bstring` | a UTF-8 string with `\x` escapes of non-UTF binary data
 `enum` | a string representing an enumeration value defined outside the scope of ZNG
 `ip` | a string representing an IP address in [IPv4 or IPv6 format](https://tools.ietf.org/html/draft-main-ipaddr-text-rep-02#section-3)
+`port` | XXX - to be filled in during PR review          |
```

Comment:
I noticed there's an entry for the `port` type in the section 3.1.1 Typedefs table, and it seems to be in the code, but it wasn't mentioned in this table here. I'm hoping someone who knows the code could plug in the correct suggestion. @alfred-landrum sounded like he's game to dig this up.